ani-cli

```diff
@@ -1,6 +1,13 @@
 #!/bin/sh
 
-version_number="4.6.1"
+version_number="4.6.2"
+
+# TODO: implement caching => @Derisis13 :point_right: :point_left:
+cleanup() {
+    rm "$HOME"/.cache/ani-cli/images/* >/dev/null 2>&1
+    ueberzug cmd -s "$ANI_CLI_UEBERZUG_SOCKET" -a exit >/dev/null 2>&1
+}
+trap cleanup EXIT INT TERM
```

Comment:
function definition should be somewhere else in the script and separate from the `trap` command